lib/bundler/definition.rb

```diff
@@ -7,7 +7,7 @@ module Bundler
   class Definition
     include GemHelpers
 
-    attr_reader :dependencies, :platforms, :ruby_version, :locked_deps
+    attr_reader :dependencies, :platforms, :ruby_version, :locked_deps, :update_opts
```

Comment:
yeah, agree - that's probably going to be renamed anyway though, i've pushed more functionality into that internal class and it needs a better name. I'm thinking DependencySearch so far. #namingishard